sklearn/tests/test_common.py

```diff
@@ -466,6 +466,8 @@ def test_estimators_do_not_raise_errors_in_init_or_set_params(Estimator):
     "ClassifierChain",
     "ComplementNB",
     "CountVectorizer",
+    "DecisionTreeClassifier",
+    "DecisionTreeRegressor",
     "DBSCAN",
```

Comment:
Not sure why these are added back?

```suggestion
```

Reply:
Sorry for that, I just made a mistake while updating the branch with the last changes in main. Many thanks for suggesting and committing the fix @Micky774 @glemaitre